README.md

```diff
@@ -1,45 +1,221 @@
 
+Key characteristics of knowledge graphs include:
 
-## Intro - What is a Decentralized Knowledge Graph (DKG)
+- Prioritization of data connections, treating them as "first-class citizens" within the realm of linked data.
+- Designed to accommodate data from diverse sources, supporting a variety of formats.
+- Their data models are flexible and easily extendable, ready to adapt to the evolving nature of data and its relationships.
 
+While knowledge graphs are typically deployed within the boundaries of a single organization to capture knowledge from various internal and external sources, we conceptualize them more broadly. We define the **Decentralized Knowledge Graph (DKG)** as a globally-shared, common infrastructure for data exchange, benefiting both organizations and individuals.
 
-There are many available definitions of a knowledge graph, therefore we will present a simplified one focused on usability, rather than completeness. The purpose of this introduction is not to be a comprehensive guide for knowledge graphs, however it aims to get you started with the basics.
+Unique attributes of the DKG are:
+
+- Empowerment of decentralized applications (Dapps) with search, integration, analytics, AI, and ML capabilities across a wide range of data sources, including blockchains, IPFS, enterprise systems, web services, and personal devices.
+- Elimination of the need for a central authority by leveraging decentralized infrastructure.
+- Facilitation of permissionless PUBLISH and QUERY operations within the public network.
+- Utilization of decentralized identity and Verifiable Credentials for access control, enabling references to private data.
+
+This expansive approach enhances accessibility and flexibility, positioning the DKG as a powerful tool in the evolving digital landscape.
+
+
+</details>
+
+<details>
+<summary>
+
+### <b>The OriginTrail DKG Architecture</b>
+</summary>
+
+<br/>
+
+The OriginTrail Decentralized Network implements the DKG according to the OriginTrail protocol.
+
+It is:
+
+* **a permissionless network** - anyone can run OriginTrail nodes
+* **a multi-chain data exchange network** - connects to several blockchains (currently Ethereum and xDai with more integrations upcoming such as with Polkadot)
+* **designed for off-chain data exchange using standardized data models** (GS1 & W3C standards and recommendations)
+* **public open source software**
+* **infrastructure for knowledge marketplaces & tenders** - more info [here](https://www.youtube.com/watch?v=4uCxYGRh5fk)
```

Comment:
This video is quite old, I would suggest to remove it

Reply:
This whole part is outdated, I haven't updated it yet